ONSdigital/sdc-global-design-patterns
# 1950 definitions

**Prerequisites**
1. Run yarn & yarn start from terminal in the project root.

**How to test - step 1**
1. From the left hand navigation find the Definition component and click.
2. Confirm the component meets the required design and behaviour.

**Step 2**
1. Run yarn test.
2. Confirm all tests pass from Karma console output.
